app/Services/AccessPointsService.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Services;
+
+use App\Plugin;
+use Illuminate\Support\Facades\Cache;
+
+class AccessPointsService {
+    
+    private const CACHE_KEY = 'access_points';
+    
+    public const /*array*/ CORE_ACCESSPOINTS = [
+        "Default" => [
+            "label" => "main.user.accesspoints.default",
+            "path" => "/",
+        ],
+    ];
+    
+    public function clearCache(): void {
+        Cache::forget(self::CACHE_KEY);
+    }
+
+    public function get(): array {
+        return Cache::rememberForever(self::CACHE_KEY, function() {
+            $accesspoints = self::CORE_ACCESSPOINTS;
+            $accesspoints = array_merge($accesspoints, self::loadAccessPointsFromPlugins());
```

Comment:
Shouldn't this be `$this->loadAccessPointsFromPlugins()` instead of `self::loadAccessPointsFromPlugins()`?

Reply:
Yeah you are right. I'll fix it.

Weird part is, I tested it and it was working. 
In PHP it seems this is working, too 😶
```php 
<?php
// example code
class SelfTest {

    public function do(){
        self::stuff();
    }

    public function stuff(){
        echo "STUFF HAPPENDED";
    }
}


new SelfTest()->do();
// Prints "STUFF HAPPENDED";
```

Reply:
Yes, this is a relict from PHP < 5 that did not have a static keyword. But: this should raise a deprecation warning in PHP8 and throw an error in PHP9.
But my IDE did not show any warnings :shrug: